SPECS/prometheus-node-exporter/prometheus-node-exporter.spec

```diff
@@ -107,6 +107,9 @@ getent passwd 'prometheus' >/dev/null || useradd -r -g 'prometheus' -d '%{_share
 %dir %attr(0755,prometheus,prometheus) %{_sharedstatedir}/prometheus/node-exporter
 
 %changelog
+* Tue Mar 29 2022 Matthew Torr <matthewtorr@microsoft.com> - 1.3.1-7
```

Comment:
Please also increment the `Release` tag value.

Reply:
Good catch, thank you.